components/collection/drop/GenerativeLayout.vue

```diff
@@ -13,7 +13,10 @@
               <div class="font-bold is-size-5 mb-4 capitalize">
                 {{ $t('tooltip.created') }}
               </div>
-              <CollectionDropCreatedBy :address="address" />
+              <CollectionDropCreatedBy
+                :address="address"
+                :show-popover="false"
```

Comment:
why change the popover?

Reply:
to fix props required from `CollectionDropCreatedBy` component